<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VCF</class>
 <widget class="QDialog" name="VCF">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VCF</string>
  </property>
  <widget class="QPushButton" name="convert">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Convert</string>
   </property>
  </widget>
  <widget class="QPushButton" name="file_open">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>File open</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>431</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Glycine max accession Williams genome assembly v1.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Glycine max accession Williams 82 genome assembly v2.0</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_result">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_output">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
